scripts/memfault/mds_ble_gateway.py

```diff
@@ -0,0 +1,399 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+import logging
```

Comment:
pc_ble_driver will soonbe deprecated

Reply:
@kenr confirm

Reply:
Do we have replacement for it?

Reply:
Is there any replacement planned?

Reply:
It is planned but it is not yet ready as far as I know.

Reply:
Anyway pc_ble_driver will be still available as python package right? Can we use it here since we don't have any replacement now?

Reply:
Let's use what's available now, and consider migrating to the new tool once it's available.